src/themes/move-theme/typography.js

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
As mentioned above, I recommend creating an "uber-theme" folder, with both the light and dark themes being maintained there. This file could be moved into that folder. Although we're calling this "move theme", it's an Uber theme. We're probably OK to rename it.